etl/steps/data/garden/oecd/2024-08-21/official_development_assistance.meta.yml

```diff
@@ -0,0 +1,1015 @@
+
+  processing_per_capita: |-
+    We calculated this indicator by dividing by the population of the donor country or regional aggregation. The population estimate is available in the OECD dataset.
+
+  processing_per_capita_2: |-
+    We calculated this indicator by dividing by the population of the recipient country. We use a long-run population dataset [mantained by Our World in Data](https://ourworldindata.org/population-sources).
+
+  processing_share_gini: |-
+    We calculated this indicator by dividing by the gross national income (GNI) of the donor country regional aggregation. The GNI estimate is available in the OECD dataset.
+
+    The OECD provides indicators as a share of GNI, but only until 2017 for net flows. We calculated this to have the entire series.
+
+  processing_oda_net_disbursements: |-
+    We have combined net disbursements aid data from the [DAC1: Flows by donor (ODA+OOF+Private) dataset](https://data-explorer.oecd.org/vis?fs[0]=Topic%2C1%7CDevelopment%23DEV%23%7COfficial%20Development%20Assistance%20%28ODA%29%23DEV_ODA%23&pg=0&fc=Topic&bp=true&snb=20&df[ds]=dsDisseminateFinalDMZ&df[id]=DSD_DAC1%40DF_DAC1&df[ag]=OECD.DCD.FSD&df[vs]=1.2&dq=DAC...1140%2B1160..Q.&lom=LASTNPERIODS&lo=10&to[TIME_PERIOD]=false) with the [DAC2A: Aid (ODA) disbursements to countries and regions dataset](https://data-explorer.oecd.org/vis?fs[0]=Topic%2C1%7CDevelopment%23DEV%23%7COfficial%20Development%20Assistance%20%28ODA%29%23DEV_ODA%23&pg=0&fc=Topic&bp=true&snb=20&df[ds]=dsDisseminateFinalDMZ&df[id]=DSD_DAC2%40DF_DAC2A&df[ag]=OECD.DCD.FSD&df[vs]=1.1&dq=.DPGC.206.USD.Q&lom=LASTNPERIODS&lo=5&to[TIME_PERIOD]=false) to add aid given by multilateral organizations and grants given by private organizations.
+
+  processing_oda_by_sector: |-
+    <% if sector == "Non-humanitarian aid" %>
+    We calculate non-humanitarian aid as the difference between total aid and humanitarian aid.
+    <%- endif -%>
+
+  processing_level_by_sector: &processing_level_by_sector |-
```

Comment:
not urgent, but I generally like to have these fields within a dedicated entry. E.g. `definitions.others` or something like that.

no need to implement for this pr.

Reply:
At least these are included within `definitions`. Do you prefer to have them all inside `definitions.others?`

Reply:
i generally do so, it's personal preference. Just because there is often times `definitions.common`, and I feel like if I have lots of things under `definitions` this one might be lost